.nais/naiserator-dev.yaml

```diff
@@ -63,4 +63,5 @@ spec:
       value: Q1_isdialogmelding
     - name: MOTTAK_QUEUE_UTSENDING_QUEUENAME
       value: QA.Q414.IU03_UTSENDING
-
+    - name: FASTLEGEREST_CLIENT_ID
+      value: "api://dev-fss.teamsykefravr.fastlegerest/.default"
```

Comment:
Det som er der nå er syntax for scope i token.
Hvis dette fungerer så tror jeg det skal være:
`dev-fss.teamsykefravr.fastlegerest` og tilsvarende for prod.

Reply:
Okk, da prøver jeg det 👍